src/parser/ParserDriver.cpp

```diff
@@ -137,6 +137,18 @@ void ParserDriver::addIO(std::unique_ptr<AstIO> d) {
                 return;
             }
         }
+    } else if (d->getType() == AstIoType::limitsize) {
```

Comment:
`d`? This should be refactored to be a useful name. This can wait for another PR, though, since it was not introduced by this PR.

Reply:
Indeed - this should be directive.